### PR TITLE
Fix for spelling

### PR DESCRIPTION
**Pull request by ntfshard**
# What's new

Fixed spelling in various places of code

# Verification 

- fbt build on linux machine

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/debug/bt_debug_app/bt_debug_app.c [hunk 1]
```diff
@@ -98,7 +98,7 @@ void bt_debug_app_free(BtDebugApp* app) {
 int32_t bt_debug_app(void* p) {
     UNUSED(p);
     if(!furi_hal_bt_is_testing_supported()) {
-        FURI_LOG_E(TAG, "Incorrect radio stack: radio testing fetures are absent.");
+        FURI_LOG_E(TAG, "Incorrect radio stack: radio testing fetuses are absent.");
```
Comment (GMMan) [inline applications/debug/bt_debug_app/bt_debug_app.c]: Lol really?

File: applications/services/input/input.h [hunk 2]
```diff
@@ -20,8 +20,8 @@ typedef enum {
     InputTypePress, /**< Press event, emitted after debounce */
     InputTypeRelease, /**< Release event, emitted after debounce */
     InputTypeShort, /**< Short event, emitted after InputTypeRelease done withing INPUT_LONG_PRESS interval */
-    InputTypeLong, /**< Long event, emmited after INPUT_LONG_PRESS interval, asynchronouse to InputTypeRelease  */
-    InputTypeRepeat, /**< Repeat event, emmited with INPUT_REPEATE_PRESS period after InputTypeLong event */
+    InputTypeLong, /**< Long event, emitted after INPUT_LONG_PRESS interval, asynchronous to InputTypeRelease  */
+    InputTypeRepeat, /**< Repeat event, emitted with INPUT_REPEAT_PRESS period after InputTypeLong event */
```
Comment (GMMan) [inline applications/services/input/input.h]: These comments are no longer accurate, suggest rewriting them to match actual behavior.

File: applications/debug/bt_debug_app/bt_debug_app.c (hunk 1 above)
Comment (ntfshard) [reply applications/debug/bt_debug_app/bt_debug_app.c]: Friday effect

File: applications/services/input/input.h (hunk 2 above)
Comment (ntfshard) [reply applications/services/input/input.h]: I'll try =)

**Review by skotopes — APPROVED** [review]

File: applications/services/input/input.h (hunk 2 above)
Comment (Soni495) [reply applications/services/input/input.h]: Zu

**Pull request merged**